bastion/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+use bastion::spawn;
+use lapin::{executor::Executor, ConnectionProperties};
+use std::{future::Future, pin::Pin};
+
+pub trait BastionExt {
+    fn with_bastion(self) -> Self
```

Comment:
Please add a with_bastion_executor and make with_bastion call it to match the other integrations